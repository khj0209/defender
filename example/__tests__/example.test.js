import React from 'react';
import Area from '../src/component/Area';
import renderer from 'react-test-renderer';

// This test was solely created to confirm the svgjs undefined issue was fixed.
// The core apexcharts library test suite contains the major test cases
test('Example test to fix svgjs issues', () => {
  const component = renderer.create(
    <Area></Area>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});