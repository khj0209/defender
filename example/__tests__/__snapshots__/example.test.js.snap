// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example test to fix svgjs issues 1`] = `
<div
  className="area"
>
  <div />
</div>
`;

exports[`Link changes the class when hovered 1`] = `
<div
  className="area"
>
  <div />
</div>
`;
